.travis.yml

```diff
@@ -24,3 +25,6 @@ jobs:
     - stage: Integration Tests
       name: "Browser Tests"
       script: make run_browser_tests
+
+after_success:
+    - .scripts/travis/push-image.sh reviewer_server
```

Comment:
Where is the command that pulls the submodule?

Reply:
Travis CI does this automatically, but if you wanted to use `.scripts` locally it could be added to the Makefile

Reply:
That would be weird however, because `make ...` will potentially reset your local changes e.g. switching to a branch of that submodule to test a new script